README.md

```diff
@@ -126,6 +126,8 @@ The SDK adds these fields to all telemetry:
 | `browser.mobile` | stable | static | [NavigatorUAData: mobile](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorUAData/mobile) | true |
 | `browser.language` | stable | static | [Navigator: language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) | "fr-FR" |
 | `browser.touch_screen_enabled` | stable | static | [Navigator: maxTouchPoints](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints) | true |
+| `device.type` | custom | static | Best guess of device type | "desktop", "mobile", "tablet", etc. |
+| `network.effectiveType` | custom | static | [NetworkInformation: effectiveType](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType). Best guess of user's "effective network type", which is based on their overall network speed. Only available on Chromium devices, and only computed once when the SDK is initialized. | "slow-2g", "2g", "3g", "4g" |
```

Comment:
is there any appetite for including the [downlink](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/downlink) property or the `rtt` as well? 

also chromium only, but can provide some extra granularity when diagnosing situations


![Screenshot 2024-05-21 at 11 32 39 AM](https://github.com/honeycombio/honeycomb-opentelemetry-web/assets/14351110/57ffe0bb-be83-4fd6-92d3-d97156faf2c1)
Screenshot from our homegrown, pre-otel instrumentation. Took inspiration from the [`web-vitals-reporter`](https://github.com/treosh/web-vitals-reporter?tab=readme-ov-file#getdeviceinfo) package

Reply:
I think we should leave it out, at least for now/until we get a request for it. 

I think it will also help to minimize the number of chrome-only fields, and this one in particular doesn't add a whole lot on top of `connection.type`. I also tend to lean with Mozilla and Apple on the privacy concerns on this one. 

It's also very easy for any customer who wants this to add their own attribute on their end, without needing us to track it by default.